apps/full-site-editing/full-site-editing-plugin/donations/blocks/donations/controls.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+// eslint-disable-next-line wpcalypso/import-docblock
+import { ExternalLink, PanelBody, ToggleControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { InspectorControls } from '@wordpress/block-editor';
+
+const Controls = ( props ) => {
+	const { attributes, setAttributes, siteSlug } = props;
+	const { showMonthly, showAnnually, showCustom } = attributes;
+	return (
+		<InspectorControls>
+			<PanelBody title={ __( 'Settings', 'full-site-editing' ) }>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **42** times:
`translate( 'Settings', { context: 'NewDash Page Title'} )` *ES Score: 8*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/donation/customize-messages)